htdocs/core/lib/files.lib.php

```diff
@@ -2523,8 +2523,14 @@ function dol_compress_dir($inputdir, $outputfile, $mode = "zip", $excludefiles =
  */
 function dol_most_recent_file($dir, $regexfilter = '', $excludefilter = array('(\.meta|_preview.*\.png)$', '^\.'), $nohook = false, $mode = '')
 {
+	global $conf;
 	$tmparray = dol_dir_list($dir, 'files', 0, $regexfilter, $excludefilter, 'date', SORT_DESC, $mode, $nohook);
-	return isset($tmparray[0])?$tmparray[0]:null;
+	//return isset($tmparray[0])?$tmparray[0]:null;
+	if(!empty($conf->global->MAIN_EMAIL_ATTACH_ALL_FILES)) {
```

Comment:
To avoid noise due to code style difference, can you enable the precommit git hook as explained here in your IDE:
https://wiki.dolibarr.org/index.php?title=Environment_and_development_tools#Install_Pre-commit_git_hook
If you do this, such errors reported by our CTI will be automatically fixed during your local commit...

Reply:
Hi. Fixed.